murdock/tests/test_api.py

```diff
@@ -394,7 +394,7 @@ def test_delete_jobs(remove, result, code):
     if result:
         assert response.json() == [job.dict(exclude_none=True) for job in result]
     else:
-        assert response.json() == {"detail": "Found no finished job to remove"}
+        assert response.json() == []
```

Comment:
Same there's no need to check if result is not empty, so the if/else whole block would become:
```python
assert response.json() == [job.dict(exclude_none=True) for job in result]
```

Reply:
yup